openapi.yml

```diff
@@ -0,0 +1,163 @@
+openapi: 3.0.0
+info:
+  version: '1.0'
+  title: Manager of Openflow pipelines
+  description: Kytos NApp to manage pipelines on the topology
+servers:
+  - url: /api/kytos/of_multi_table
+paths:
+  /v1/pipeline:
+    get:
+      summary: Get pipelines
+      description: Get a list of pipelines
+      operationId: list_pipelines
+      responses:
+        '200':
+          description: OK
+    post:
+      summary: Add pipeline
+      operationId: add_pipeline
+      requestBody:
+        description: Creates a new pipeline.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/NewPipeline'
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Request do not have a valid JSON.
+        '415':
+          description: The request body mimetype is not application/json.
+
+  /v1/pipeline/{pipeline_id}:
+    get:
+      summary: Get a pipeline
+      description: Get a single pipeline
+      operationId: get_pipeline
+      parameters:
+        - name: pipeline_id
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
```

Comment:
Shouldn't this be returning a instead of `Pipeline`? or are you handle this in a subsequent PR?

Reply:
Yes, the next is to add the collection to mongo.

Reply:
Right.